app.ts

```diff
@@ -0,0 +1,89 @@
+import { createServer, IncomingMessage, ServerResponse } from 'http';
+import * as fs from 'fs';
+
+const host: string = '127.0.0.1';
+const port: number = 3000;
+const path: string = './newFile.txt';
```

Comment:
Move constants to the separate file for constants.

Reply:
I've made all the corrections, tell me please if it's correct to write the explanation in one commit or I should add each separate commit to an appropriate file. Thank you